playbooks/adhoc/uninstall.yml

```diff
@@ -41,7 +42,7 @@
         - origin-node
 
     - yum: name={{ item }} state=absent
-      when: not is_atomic
+      when: is_atomic == "False"
```

Comment:
hmm, not is_atomic seems to be more intuitive for me (and I believe it is considered more "pythonic", but I wouldn't be surprised to find out it improves the readability of the error. Also testing against a string "False" seems quite odd.

If having an issue with the fact being defined properly, you could do something like this for is_atomic:
```
is_atomic: "{{ (ostree_output.rc == 0) if rc in ostree_output else False }}"